rtt/typekit/BoolTypeInfo.hpp

```diff
@@ -49,10 +49,10 @@ namespace RTT
         /**
          * Write boolean as 'true' or 'false'.
          */
-        struct BoolTypeInfo: public TemplateTypeInfo<bool,true>
+        struct BoolTypeInfo: public StdTypeInfo<bool>
         {
             BoolTypeInfo() :
-                TemplateTypeInfo<bool,true> ("bool")
+                StdTypeInfo<bool> ("bool")
             {
```

Comment:
I'm afraid that if someone has a component that use rtt from a package (binary) he/her will have to rebuild it, because the rtt-typekit changed ... but, if you tested it and it's ok?

Will you do a new release?